lib/misc/blocks.js

```diff
@@ -16,21 +16,21 @@ function isBlank ({line, scope}, allowDocstrings = false) {
       return true
     }
   }
-  return line.match(/^\s*(#.*)?$/)
+  return /^\s*(#.*)?$/.test(line)
 }
-function isEnd ({ line }) {
+function isEnd ({ line, scope }) {
+  for (const s of scope) {
+    if (/\bstring\.multiline\.end\b/.test(s)) {
+      return true
+    }
+  }
   return line.match(/^(end\b|\)|\]|\})/)
 }
 function isCont ({ line }) {
   return line.match(/^(else|elseif|catch|finally)\b/)
 }
 function isStart (lineInfo) {
-  for (const s of lineInfo.scope) {
-    if (/\bstring\b/.test(s)) {
-      return false
-    }
-  }
```

Comment:
um, so if we remove this condition check, the code like below won't get selected as expected again:
```jl
ret = """
multiline string
...
""" |> somefunc
```

But I've not been able to figure out how to handle this while correctly run pieces like

```jl
str = "a line containing string scope"
str2 = "another line containing string scope"
...
```

Reply:
Urgh, true. I only tested
```
ret = """
  multiline string
  aksjdha
""" |> somefunc
```
Will take another look...

Reply:
(I'm trying not to say let's use CSTParser for this too ...)

Reply:
Those cases work now.